### DISCUSSION TITLE
Basics from newbie

### DISCUSSION THREAD

**Discussion by LukyGuyLucky** [c3lang/c3c#1120] [Q&A] [answered]
When using precompiled binary package under windows 10 64 bit ,is it mandatory to download VC17 stuffs by executing msvc_build_libraries.py?
I did that but it seems it does't matter when I compile the first Hello World test.

Was I missing something or under what situation vc17 stuffs is mandatory required?

---Win10 ,64 bit,with VS 2019 community installed already,but not VS 2022(VC17).

**Comment by lerno** [comment, marked as answer]
If you already have MSVC installed, it should automatically work. Could you show me what errors you get?

**Reply by LukyGuyLucky** [reply]
No,no errors got.I mean I have VC16 rather than VC17 installed but it still works.So there could be a misunderstanding when one first read "install VC17" is mandatory.

**Comment by LukyGuyLucky** [comment]
Solved.
